src/app/modules/project/add/manual/manual.component.html

```diff
@@ -1,118 +1,131 @@
+  <div class="offset-2 col-8">
+    <h1>Add new project</h1>
+    <form [formGroup]="newProjectForm" class="project-form">
+
+      <div class="row">
+        <div class="col-6">
+          <div>
+            <h3>Project name*</h3>
+            <input placeholder="Name of your project" formControlName="name" required />
+          </div>
+        </div>
+
+        <div class="col-6">
+          <div>
+            <h3>Project link*</h3>
+            <input placeholder="Link to your project, https://github.com/fontys/projectname" formControlName="uri" required />
+          </div>
+        </div>
+      </div>
+
+      <div class="row">
+        <div class="col-12">
+          <div>
+            <h3>Short description*</h3>
+            <input placeholder="Fill in a short description of your project" formControlName="shortDescription" required />
+          </div>
+        </div>
+      </div>
+
+      <div class="row">
+        <div class="col-12">
+          <div>
+            <p class="form-footnote">Describe your project in one sentence or 170 characters.</p>
+          </div>
+        </div>
+      </div>
+
+      <div class="row">
+        <div class="col-12">
+          <div class="long-description-editor">
+            <h3>Description*</h3>
+            <quill-editor class="dex-quill-editor" [styles]="{height: '350px'}" [modules]="modulesConfigration"
+              formControlName="description" placeholder="">
+            </quill-editor>
+          </div>
+        </div>
+      </div>
+    </form>
+
+    <form [formGroup]="newCollaboratorForm" class="project-form">
+      <div class="row">
+        <div class="col-12">
+          <h3 class="title-spacing">Add project collaborators</h3>
+          <div class="row">
+
+            <div class="col-5">
+              <div>
+                <h3>Collaborator full name</h3>
+                <input placeholder="Name of the collaborator" formControlName="fullName" required />
+              </div>
+            </div>
+
+            <div class="col-5">
+              <div>
+                <h3>Collaborator role</h3>
+                <input placeholder="Role of the collaborator" formControlName="role" required />
+
+              </div>
+            </div>
+
+            <div class="col-2 add-collaborator-col">
+              <button type="button" class="btn btn-icon btn-primary add-collaborator-button" aria-label="settings"
+                (click)="onClickAddCollaborator()">
+                +
+              </button>
+            </div>
+
+            <div class="offset-5 col-5">
+              <p class="form-footnote">ex: developer, designer, etc...</p>
+            </div>
+
+          </div>
+        </div>
+      </div>
+    </form>
+
+    <div class="row collaborator-overview" *ngIf="collaborators.length > 0">
+
+      <div class="col-12">
+        <h3>Collaborators</h3>
+      </div>
+
+      <div *ngFor="let collaborator of collaborators" class="col-12 collaborator-item">

```

Comment:
```suggestion
      <div *ngFor="let collaborator of collaborators" class="col-6 collaborator-item">

```

Keeping this on col-12 would mean every contributors gets it's own line. For big projects, with maybe 20-30 collaborators, you will get a very big page which can be avoided by simply having 2 rows of contributors
Also looks cleaner in my opinion